<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>447</width>
    <height>242</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="name">
             <property name="text">
              <string>name</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="IP">
             <property name="text">
              <string>IP</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="login">
             <property name="text">
              <string>USER</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="password">
             <property name="text">
              <string>password</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="port">
             <property name="text">
              <string>port</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="1">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLineEdit" name="NameEdit">
             <property name="text">
              <string>fn1132_2020</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="IPEdit">
             <property name="text">
              <string>195.19.32.74</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="LoginEdit">
             <property name="text">
              <string>student</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="PasswordEdit">
             <property name="text">
              <string>bmstu</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="PortEdit">
             <property name="text">
              <string>5432</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="PushButton">
         <property name="text">
          <string>ok</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
